h5py/_hl/files.py

```diff
@@ -429,6 +433,8 @@ def __init__(self, name, mode='r', driver=None, libver=None, userblock_size=None
         track_order
             Track dataset/group/attribute creation order under root group
             if True. If None use global default h5.get_config().track_order.
+        track_times
+            Store timestamps for the root group in the file.
```

Comment:
I should have seen this the first time around, sorry
```suggestion
        track_times: bool or None, default: False
            If True, store timestamps for this group in the file.
            If None, fall back to the default value.
```